### ISSUE TITLE
Discuss whether to provide a stablecoin-backed base fee burning model.

### ISSUE THREAD

**Issue by ljboy123** [tronprotocol/tips#405]
## Simple Summary

Try using a more stable fee payment model. Suppress the harm caused by currency price fluctuations to users.

## Motivation 

This discussion only represents some personal thoughts.

Burn $1 worth of trx to produce 1 UT Trc10
Burning UT cannot produce equivalent TRX
Establish the corresponding relationship between UT, energy and bandwidth.
1 TRX = $0.062
1 BANDWIDTH = 0.000062 UT
1 ENERGY = 0.00001736 UT
A single usdt transfer will consume 0.2728 UT

Considering that this might reduce the value of trx's existence.
But widespread use of UT will go beyond paying for energy and bandwidth.
There may also be other ways of value empowerment and value storage in UT.
Additional use-value of UT will bring additional TRX burn.
This can bring a stable user volume to Tron.

When the value of trx increases, the base fee used will also increase.
Rising base fees will reduce user demand and cause users to choose cheaper blockchains.
Some users choose to use TRON because the basic fee of Ethereum is too expensive.
But any investor who holds the blockchain's native token hopes to gain value.

This creates a very bad cycle
Investors expect to burn more tokens to increase the value -> basic usage fees increase -> users transfer other blockchains | reduce the number of times users use blockchains -> reduce basic usage fees -> burned native tokens Decline -> Decrease investor confidence.
If using UT burn base fee
Investors expect to burn more native tokens to increase the value -> the basic usage fee will remain unchanged -> the blockchain users and usage times will remain unchanged -> no need to reduce the basic usage fee -> the burned UT remains unchanged (conversion The burning native Token drops) -> less impact on investors

When the value of 1UT is > $1, burn more Trx to produce UT
When the value of 1UT is < $1, the stock of UT will continue to decrease due to burning. It is also possible to burn more UT by increasing the base usage fee.

**Comment by joker-gdb** [comment]
Looks like luna～

**Comment by ljboy123** [comment]
> Looks like luna～
Yes, in LUNA UST can produce LUNA. A run on UST will lead to a collapse in the value of LUNA. However, a run on UT will lead to a collapse in the value of UT, but will not lead to a collapse in the value of TRX, because UT cannot produce TRX. After the UT price collapses, it will be pulled back to the original price by doubling the burning. Users who sell UT at a low price will become the main force in burning TRX.

**Comment by ljboy123** [comment]
> Looks like luna～

Yes, in LUNA UST can produce LUNA. A run on UST will lead to a collapse in the value of LUNA. However, a run on UT will lead to a collapse in the value of UT, but will not lead to a collapse in the value of TRX, because UT cannot produce TRX. After the UT price collapses, it will be pulled back to the original price by doubling the burning. Users who sell UT at a low price will become the main force in burning TRX.

**Comment by ljboy123** [comment]
> Looks like luna～

That is to say, there is a lot of risk in holding UT. Buying UT at a low price is an investment with income, and selling UT at a low price is a loss. When the price of TRX rises, there will be a large demand for UT due to the TRX-UT trading pair. And UT. Can only be produced through TRX. TRX deflation will intensify. Again, TRX rose. When the price of TRX falls, the price of UT may drop due to panic. The falling UT will be burned in the form of user payment basis. If UT is not worth $1. The base fee requires a $1 contract transaction and will burn $1 worth of UT. Users who sell UT at a low price are at a loss. The value of the losses will be captured by Tron and other investors. I don't know if this understanding is flawed.

**Comment by joker-gdb** [comment]
Prove you are right, maybe TRON set sail again!

**Comment by ljboy123** [comment]
> Prove you are right, maybe TRON set sail again!

Thank u
